GrocifyApp.UI/src/components/Login.jsx

```diff
@@ -1,16 +1,27 @@
 import React, { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
+import { ReactSVG } from 'react-svg';
+import ReactLogo from '../assets/logo_with_text.svg';
+import ArrowIcon from '../assets/arrow-ic.svg';
+import './Login.jsx.scss';
+import ApiEndpoints from '../consts/ApiEndpoints';
+import { GenericConsts, LoginConsts } from '../consts/ENConsts';
 
 const LoginForm = () => {
     const navigate = useNavigate();
     const [email, setEmail] = useState('');
     const [password, setPassword] = useState('');
+    const [errorMessage, setErrorMessage] = useState('');
 
-    const handleLogin = async () => {
+    const handleLogin = async (event) => {
```

Comment:
how this event works?

Reply:
javascript. with this event, you can use event.target and it gives you the element that was clicked